cadccutout/setup.cfg

```diff
@@ -34,13 +34,13 @@ long_description = Library written in Python to be integrated into different ser
 author = Canadian Astronomy Data Centre
 author_email = cadc@nrc-cnrc.gc.ca
 license = AGPLv3
-url = http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/
+url = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/
 edit_on_github = False
 github_project = opencadc/cadctools/cadccutout
-install_requires = astroquery>=0.3.8 astropy>=2.0.9 regions>=0.3 aenum
-dev_install_requires = pytest>=3.9.0 pytest-cov>=2.5.1 coveralls>=1.5.1 flake8>=3.5.0 pytest-arraydiff memory_profiler
+install_requires = astropy regions aenum fitsio@git+https://github.com/at88mph/fitsio.git@full-merge#egg=fitsio
```

Comment:
Temporary?

Reply:
Yes.  There is still one outstanding PR to the `fitsio` developers.